acceptance/Gemfile

```diff
@@ -1,6 +1,6 @@
 source ENV['GEM_SOURCE'] || "https://rubygems.org"
 
-gem "beaker", "~> 1.17"
+gem "beaker", "~> 2.23"
```

Comment:
yep I'm fine using pessimistic pin, I copy/pasta'ed from stable and wanted to make sure we're using a known good version of beaker.